clients/destination.go

```diff
@@ -300,7 +301,20 @@ func (c *DestinationClient) Write2(ctx context.Context, tables schema.Tables, so
 	}
 	_, err = saveClient.CloseAndRecv()
 	if err != nil {
-		return fmt.Errorf("failed to CloseAndRecv client: %w", err)
+		st := status.Convert(err)
+		errorWithDetails := st.Message()
+		if len(st.Details()) > 0 {
+			errorWithDetails += "\nDetails:\n"
```

Comment:
can we just utilize [prototext](https://pkg.go.dev/google.golang.org/protobuf/encoding/prototext)?

Reply:
We can try good catch. Please note this is only one example on how to report the errors. We might want to do something else than strings concatenation